css/style.css

```
@@ -439,6 +437,7 @@ img {
   position: absolute;
   width: 138px;
   left: 571px;
+  order: 1;
 }
```

Есть один очень расхожий трюк, как центрировать блок расположенный абсолютом:
```suggestion
  position: absolute;
  left: 50%;
  transform: translateX(-50%);
}
```
не забудь задать родительскому блоку `position: relative;`. Будет центрироваться и без этого, но так лучше